mod-di-converter-storage-server/src/main/java/org/folio/dao/association/ProfileWrapperDao.java

```diff
@@ -51,4 +51,12 @@ public interface ProfileWrapperDao {
    * @return future with founded ProfileWrapper
    */
   Future<List<ProfileWrapper>> getWrapperByProfileId(String profileId, ProfileType profileType, String tenantId);
+
+  /**
+   * Get count of the strings in the table
+   * @param tenantId - tenant id
+   * @param tableName - table name
+   * @return future with count of string in the table
+   */
+  Future<Integer> getLinesCount(String tenantId, String tableName);
```

Comment:
since there is tableName as one of the parameters, it seems this method should be in a separate utility class. it is not specific to the profile_wrappers table.

Reply:
I first wanted to make this method more specific, but then I realized that it is quite general, and in general, I hope in the next version we will remove all this migration.